Sources/BraintreeCard/BTConfiguration+Card.m

```diff
@@ -1,29 +0,0 @@
-#import "BTConfiguration+Card.h"
-
-// Swift Module Imports
```

Comment:
What does `Swift Module Imports` mean?

Reply:
"Import all Braintree modules that are written in Swift?"

Reply:
`Swift Module Imports` is the header used for any of the imports of the Swift modules (Core or DataCollector) in any of the Obj-C files so we can hopefully easily sepearate where we are importing Obj-C vs Swift things in any of the Obj-C implementation files. If there is an easier way to separate these to make it clearer, happy to update them!

Reply:
Ah ok - I think what confused me is this comment looked only like it it applied to the CocoaPods line below, instead of the entire if else block.

Also I think the word "module" has several meanings. I thought first of the `module style imports` (@ syntax).

Maybe we can specify `// Import Braintree's Swift files` or something?

Reply:
For sure! We can do whatever will make it most clear where we are separating out imports for Obj-C modules/files vs Swift modules/files. We also currently have this for Obj-C imports ([example here](https://github.com/braintree/braintree_ios/blob/next-major-version/Sources/BraintreePaymentFlow/BTPaymentFlowClient.m#L4)) `// Objective-C Module Imports`. Should we update that as well?

Reply:
Matt and I were talking about this earlier. What do we think about replacing these:
```
// Objective-C Module Imports
// Swift Module Imports
```

With:
```
Option 1:
// Objective-C File Imports
// Swift File Imports

Option 2:
// MARK: - Objective-C File Imports
// MARK: - Swift File Imports

Option 3:
// Objective-C Imports for Package Managers
// Swift Imports for Package Managers
```
Thoughts on any of these or something else entirely?

Reply:
I like option #2! But I think it's also easy enough to decipher the ObjC vs Swift file imports since the swift ones all end in `-Swift.h`. But whatever you guys think is more readable works for me 👍